### DISCUSSION TITLE
Increase recording time

### DISCUSSION THREAD

**Discussion by coderunway** [roflcoopter/viseron#688] [Q&A] [answered]
Currently, all recordings are of 5 to 10 seconds. As a result, it ends up creating a large list of small files. Is there a config parameter that can increase this time or file size?

**Comment by roflcoopter** [comment, marked as answer]
It should record for as long as there is activity (detected object or motion depending on your config)

If you want to increase the minimum recording time you can set `idle_timeout` under `recorder` for the camera to specify for how long to keep recording after activity has stopped.
